modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -34,14 +34,16 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
-  profiles:
-  - AffinityAndTaints       <1>
-  - TopologyAndDuplicates   <2>
-  - LifecycleAndUtilization <3>
+  profileCustomizations:
+    podLifetime: 30m        <1>
+  profiles:                 <2>
+  - AffinityAndTaints
+  - TopologyAndDuplicates   <3>
+  - LifecycleAndUtilization
 ----
-<1> Enable the `AffinityAndTaints` profile, which evicts pods that violate inter-pod anti-affinity, node affinity, and node taints.
-<2> Enable the `TopologyAndDuplicates` profile, which evicts pods in an effort to evenly spread similar pods, or pods of the same topology domain, among nodes.
-<3> Enable the `LifecycleAndUtilization` profile, which evicts long-running pods and balances resource usage between nodes.
+<1> Optional: Enable a custom pod lifetime value for the `LifecycleAndUtilization` profile. The default pod lifetime is 24 hours.
+<2> Add one or more profiles to enable. Available profiles: `AffinityAndTaints`, `TopologyAndDuplicates`, `LifecycleAndUtilization`, `SoftTopologyAndDuplicates`, `EvictPodsWithLocalStorage`, and `EvictPodsWithPVC`.
+<3> Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`, because they will conflict with each other.
```

Comment:
@damemi Can you give me any more info about not setting these two profiles at the same time? Like what behavior would happen if you did?

I can see that it is possible to set them both in the config, there aren't any errors.

Reply:
the config won't immediately give an error if you try to set both, but the [operator will](https://github.com/openshift/cluster-kube-descheduler-operator/blob/16ec69f/pkg/operator/target_config_reconciler.go#L326)

The difference is that `SoftTopology...` includes `ScheduleAnyway` topology constraints, while `Topology..` excludes them. So setting both logically conflicts, and the operator will throw an error to alert the user of that

Reply:
s/`SoftTopologyAndDuplicates`,/`SoftTopologyAndDuplicates`